instance/migrations/0113_auto_20181129_1622.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
@giovannicimolin, since this change is still in development, you should squash these migrations into one.

Reply:
@lgp171188 ok!